packages/core/src/tools/utils/urlPolyfill.spec.ts

```diff
@@ -15,8 +24,13 @@ describe('normalize url', () => {
   })
 
   it('should keep non http url unchanged', () => {
+    expect(normalizeUrl('ftp://foo.com/my/path')).toEqual('ftp://foo.com/my/path')
+  })
+
+  it('should keep file url unchanged', () => {
     if (isFirefox()) {
-      pending('https://bugzilla.mozilla.org/show_bug.cgi?id=1578787')
+      // On firefox, URL host is empty for file URI: 'https://bugzilla.mozilla.org/show_bug.cgi?id=1578787'
+      expect(normalizeUrl('file://foo.com/my/path')).toMatch(/^file:\/\/.*\/my\/path$/)
```

Comment:
I made the test for firefox ignoring the host so that if they fix the issue the test still works

Reply:
I'd be in favour of not doing that.
It would make the test clearer to have the exact output and when the issue will be solved and after we update the firefox version, we could remove this if instead of keeping it.